ezcontent.profile

```diff
@@ -91,3 +91,19 @@ function ezcontent_themes_installed($theme_list) {
     }
   }
 }
+
+/**
+ * Alter custom block forms rendered by Block Content and Layout Builder.
+ *
+ * E.g. Alter block forms for 'block_content' and 'inline_block' plugins.
+ *
+ * @param array $form
+ *   Nested array of form elements that comprise the form.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The form state.
+ * @param string $block_type
+ *   The machine name of the custom block bundle.
+ */
+function ezcontent_block_type_form_alter(array &$form, FormStateInterface &$form_state, string $block_type) {
```

Comment:
The library should only be attached to cards block, also i believe it would be better to move these code to block level, inside cards block module.